website/static/js/filepage/revisions.js

```diff
@@ -140,7 +140,7 @@ var FileRevisionsTable = {
                     return m('.alert.alert-warning', {style:{margin: '10px'}}, model.errorMessage);
                 }
 
-                return m('table.table', [
+                return m('table.table', {style: {tableLayout: 'fixed'}}, [
```

Comment:
Yes, it is better to create one class. 
.fixedTableLayout {
   table-layout: 'fixed';
}

It can make the width the same in different columns. This class is also very likely to be used in other tables.